build_tools/benchmarks/suites/iree_benchmarks.py

```diff
@@ -0,0 +1,81 @@
+## Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Defines IREE benchmarks."""
+
+from itertools import product
+from typing import List, Tuple
+from .device_specs.linux_x86_64 import GCP_C2_STANDARD_16_DEVICE_SPEC
+from .models.model_group import MOBILE_MODEL_GROUP
+from .definitions.common import DeviceArchitecture, DevicePlatform, RANDOM_MODEL_INPUT_DATA
+from .definitions.iree import (IreeBenchmarkCompileSpec, IreeBenchmarkRunSpec,
+                               IreeCompileTarget, IreeCompileConfig,
+                               IreeRunConfig, IreeRuntimeDriver,
+                               IreeRuntimeLoader, IreeTargetBackend)
+from .id_defs import (IREE_COMPILE_CONFIG_LINUX_CASCADELAKE_ID,
+                      IREE_RUN_CONFIG_LOCAL_SYNC_ID,
+                      IREE_RUN_CONFIG_LOCAL_TASK_ID_BASE)
+
+MODULE_BENCHMARK_TOOL = "iree-benchmark-module"
+
+
+class Linux_x86_64_Benchmarks(object):
+  """Benchmarks on x86_64 linux devices."""
+
+  CASCADELAKE_CPU_TARGET = IreeCompileTarget(
+      target_architecture=DeviceArchitecture.X86_64_CASCADELAKE,
+      target_platform=DevicePlatform.LINUX_GNU,
+      target_backend=IreeTargetBackend.LLVM_CPU)
+
+  CASCADELAKE_COMPILE_CONFIG = IreeCompileConfig(
+      id=IREE_COMPILE_CONFIG_LINUX_CASCADELAKE_ID,
+      tags=["default-flags"],
+      compile_targets=[CASCADELAKE_CPU_TARGET])
+
+  @classmethod
+  def generate(
+      cls) -> Tuple[List[IreeBenchmarkCompileSpec], List[IreeBenchmarkRunSpec]]:
+    """Generates IREE compile and run specs."""
+
+    default_run_configs = cls._generate_default_run_configs()
+
+    # Generate compile specs for mobile models.
+    mobile_model_compile_specs = list(
+        IreeBenchmarkCompileSpec(compile_config=cls.CASCADELAKE_COMPILE_CONFIG,
+                                 model=model) for model in MOBILE_MODEL_GROUP)
```

Comment:
Any reason to not just use a list comprehension here?

```suggestion
    mobile_model_compile_specs = [
        IreeBenchmarkCompileSpec(compile_config=cls.CASCADELAKE_COMPILE_CONFIG,
                                 model=model) for model in MOBILE_MODEL_GROUP]
```

Reply:
For some reasons I thought that will create a list of generators. Done.